tests/kernel/cache/prj.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_ZTEST=y
 CONFIG_TEST_USERSPACE=y
+CONFIG_CACHE_MANAGEMENT=y
```

Comment:
Now we, probably, need to remove `filter: CONFIG_CACHE_MANAGEMENT`

Reply:
I don't think so, because the prj.conf sets CONFIG_CACHE_MANAGEMENT, but it might stay deselected if targets do not have CPU_HAS_DCACHE/CPU_HAS_ICACHE (which respectively make DCACHE/ICACHE selected by default, these being the dependency for CACHE_MANAGEMENT).